cranelift/native/src/lib.rs

```diff
@@ -5,8 +5,12 @@
     missing_docs,
     trivial_numeric_casts,
     unused_extern_crates,
-    unstable_features
+    // unstable_features
 )]
+/*
+    riscv64gc backend have to use  is_riscv_feature_detected which is unstable.
+*/
+#![feature(stdsimd)]
```

Comment:
This will need to be conditional on riscv64 being used.

Reply:
~~~
pub fn builder_with_options(infer_native_flags: bool) -> Result<isa::Builder, &'static str> {
~~~
I remove the infer_native_flags functionality for risc-v64 , I tried some time , but still compile not ok for stable tool chain.